pystiche_papers/johnson_alahi_li_2016/_utils.py

```diff
@@ -15,10 +15,27 @@ def postprocessor() -> transforms.CaffePostprocessing:
 
 
 def multi_layer_encoder(impl_params: bool = True,) -> enc.VGGMultiLayerEncoder:
+    r"""Multi-layer encoder based on the VGG16 architecture with the weights of caffe,
```

Comment:
Is there a reason you diverted from the scheme to simply put the name and citation in the first docstring line?